package.json

```diff
@@ -369,18 +369,18 @@
 		"test": "gulp test"
 	},
 	"devDependencies": {
-		"@types/fs-extra": "^9.0.11",
-		"@types/glob": "^7.1.3",
-		"@types/gulp": "^4.0.8",
+		"@types/fs-extra": "^9.0.12",
+		"@types/glob": "^7.1.4",
+		"@types/gulp": "^4.0.9",
 		"@types/gulp-sourcemaps": "^0.0.34",
-		"@types/js-yaml": "^4.0.1",
-		"@types/mocha": "^8.2.2",
-		"@types/node": "^14.16.0",
-		"@types/node-netstat": "^1.8.0",
+		"@types/js-yaml": "^4.0.2",
+		"@types/mocha": "^8.2.3",
+		"@types/node": "^14.17.5",
+		"@types/node-netstat": "^1.8.1",
 		"@types/ps-tree": "^1.1.0",
-		"@types/semver": "^7.3.6",
-		"@types/terser-webpack-plugin": "^5.0.3",
-		"@types/vscode": "^1.57.0",
+		"@types/semver": "^7.3.7",
+		"@types/terser-webpack-plugin": "^5.0.4",
+		"@types/vscode": "^1.58.0",
```

Comment:
If you increment the version of the VS Code type definitions, also increment the VS Code engine version (to ensure we're not trying to use types that exist only on a newer version).

Reply:
Thanks for pointing out this. I just used `npm update` and didn't know of these gotchas!